wandb/util.py

```diff
@@ -190,9 +190,13 @@ def is_tf_tensor(obj):
     return isinstance(obj, tensorflow.Tensor)
 
 
+def is_jax_tensor_typename(typename):
+    return typename.endswith('_FilledConstant') or typename.endswith('DeviceArray')
```

Comment:
Don't like the endswith check here.  There could be other objects that end with these strings.  I assume the types start with "jax"?